eq_cir_management_ui/main/routes.py

```diff
@@ -0,0 +1,39 @@
+"""Routes for the EQ CIR Management UI."""
+
+import logging
+
+from flask import (
+    Blueprint,
+    render_template,
+    request,
+)
+
+main_blueprint = Blueprint("main", __name__)
+
+logger = logging.getLogger(__name__)
+
+
+@main_blueprint.before_request
+def before_request_func() -> None:
+    """Log the request before it is processed."""
+    if request.endpoint != "health-check":
+        message = "Request received"
+        logger.info(message)
+
+
+@main_blueprint.route("/", methods=["GET"])
+def index() -> str:
+    """UI index.
+
+    :return: 200 index page.
+    """
+    return render_template("index.html")
+
+
+@main_blueprint.route("/health-check", methods=["GET"])
```

Comment:
When trying to get to this endpoint first time after starting/restarting the app I get:
```
raise NotFound() from None
werkzeug.exceptions.NotFound: 404 Not Found
```
and 
```
"GET /favicon.ico HTTP/1.1" 404
```
the url is `http://127.0.0.1:5100/favicon.ico`

Reply:
This should now be solved - have added back the static folder config, and added a specific root for /favicon.ico

Reply:
Yeah, seems to work ok now, thanks.